src/blueapi/service/main.py

```diff
@@ -167,10 +170,11 @@ def delete_submitted_task(
     response_model=WorkerTask,
     responses={status.HTTP_409_CONFLICT: {"worker": "already active"}},
 )
-def update_task(
+def set_active_task(
     task: WorkerTask,
     handler: BlueskyHandler = Depends(get_handler),
 ) -> WorkerTask:
+    """Begin the specified task."""
```

Comment:
Maybe this isn't quite optimal wording...

Reply:
How about?

```suggestion
    """Promote a task to active status, the worker should begin it as soon as possible"""
```

Reply:
Promote implies (to me) that it can displace other currently running tasks.
"Attempt to set the task to be run on the worker"?